src/components/Person/Person.tsx

```diff
@@ -80,9 +80,9 @@ export const Person = ({
           </p>
         )}
       </div>
-      {(description || !!links?.length) && (
+      {((description && layoutVariant !== 'compact') || !!links?.length) && (
         <div className="cc-person__body">
-          {description && (
+          {description && layoutVariant !== 'compact' && (
```

Comment:
You use the same check twice here... could it be worth making it more readable (and easier to update later, if the condition changes), to extract the check to a boolean variable.

```
const shouldShowDescription = description && layoutVariant !== 'compact';

// later...

{shouldShowDescription && /* ... */}
{shouldShowDescription || !!links?.length && ( /* ... */ )}

Reply:
updated :)